app/react-native/src/preview/components/OnDeviceUI/style.js

```diff
@@ -3,24 +3,51 @@ import { StyleSheet } from 'react-native';
 export default {
   main: {
     flex: 1,
+    paddingTop: 20,
+    backgroundColor: 'rgba(255, 255, 255, 1)',
+  },
+  icon: {
+    width: 20,
+    height: 20,
+    opacity: 0.5,
+  },
+  headerContainer: {
     flexDirection: 'row',
+    alignItems: 'center',
+    margin: 5,
+  },
+  headerText: {
+    marginLeft: 5,
+    fontSize: 14,
+    color: 'rgba(0, 0, 0, 0.5)',
+  },
+  overlayContainer: {
+    position: 'absolute',
```

Comment:
For consistency, you can use [StyleSheet.absoluteFillObject](https://facebook.github.io/react-native/docs/stylesheet.html#absolutefillobject) like used below in the `preview` style 😃